src/datasource.ts

```diff
@@ -270,7 +270,7 @@ export default class HeroicDatasource {
     options.headers = headers;
     options.url = this.settings.url + path;
     options.inspect = { type: "heroic" };
-    if (options.method === 'POST' && options.data) {
+    if (options.method === 'POST' && options.headers['Content-Type'] === 'application/json;charset=UTF-8') {
```

Comment:
can `application/json;charset=UTF-8` be made into a constant and reuse that here and https://github.com/spotify/spotify-heroic-datasource/blob/cf2ca8159ed1ca05cb3b476fc843ab2d260f5400/src/datasource.ts#L281 so the two don't get out of sync?

Reply:
Yep.